rack-box/scripts/install.sh

```diff
@@ -183,5 +183,5 @@ done
 # Setup the RACK dataset using the RACK CLI
 
 cd "/home/${USER}/RACK/cli/"
-python3 -m pip install ./wheels/*.whl
+python3 -m pip install --no-dependencies ./wheels/*.whl
```

Comment:
Why not `pip3 install`?

Reply:
only because I wasn't thinking about changing this line of code at all until it failed due to dependency checks. I was singularly focused on `--no-dependencies` when I visited this line. I'll make the change.

In case you wondered and didn't consult the failed build logs, the `--no-dependencies` became necessary because pip was able to tell that the version of `python3-semtk` that we'd installed was not the GitHub version, as specified in the `pyproject.toml` but it was the pre-compiled wheel file which self-descripted as version `0.1`. This flag disables all the checks. I believe that this is OK because all the checks were done at the point at which we generated the wheels.